src/kudu/consensus/raft_consensus.cc

```diff
@@ -2405,6 +2405,19 @@ string RaftConsensus::GetLeaderUuid() const {
   return GetLeaderUuidUnlocked();
 }
 
+std::pair<string, unsigned int> RaftConsensus::GetLeaderHostPort() const
```

Comment:
probably this function makes more sense in the ConsensusMetadata class.
The ConsensusMetadata Manager  is managed by TabletServer/SimpleTabletManager and
its lifetime is all of TabletServer.

So what we probably need is a way to access the cmeta_manager_  from the plugin.

Reply:
RaftConsenus has a cmeta_ member which can be used for this, let me see if it can be done.